composer/algorithms/agc/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+"""`Automatic Gradient Clipping <https://arxiv.org/abs/2102.06171>`_ Clips all gradients in model based on ratio of
```

Comment:
I think its Adaptive gradient clipping, not Automatic

Reply:
Fixed in 9992338

Reply:
Good catch, @hanlint ! I found another place where I did that too.